habitat_sim/simulator.py

```diff
@@ -247,23 +273,32 @@ def get_observation(self):
         # internally it will set the camera parameters (from the sensor) to the
         # default render camera in the scene so that
         # it has correct modelview matrix, projection matrix to render the scene
-        self._sim.renderer.draw(self._sensor_object, scene)
 
-        if self._spec.sensor_type == hsim.SensorType.SEMANTIC:
-            self._sim.renderer.readFrameObjectId(self._buffer)
-            return np.flip(self._buffer, axis=0).copy()
-        elif self._spec.sensor_type == hsim.SensorType.DEPTH:
-            self._sim.renderer.readFrameDepth(self._buffer)
-            return np.flip(self._buffer, axis=0).copy()
-        else:
-            self._sim.renderer.readFrameRgba(self._buffer)
-            return np.flip(
-                self._buffer.reshape(
-                    (
-                        self._spec.resolution[0],
-                        self._spec.resolution[1],
-                        self._spec.channels,
-                    )
-                ),
-                axis=0,
-            ).copy()
+        with self._sensor_object.rendering_target as tgt:
```

Comment:
What are people thoughts about getting rid of the clones/copies here?  They are kinda slow (especially the one on the GPU tensor) and for a lot of use-cases, they aren't needed (for RL training, we just immediately copy the image into a big tensor of images).